session/session.go

```diff
@@ -1911,6 +1911,9 @@ func (s *session) ExecuteStmt(ctx context.Context, stmtNode ast.StmtNode) (sqlex
 		}
 	})
 
+	if err := sessiontxn.WarmUpTxn(s); err != nil {
```

Comment:
Always warming up seems unnecessary. For example when it will use max_uint as ts.